### PR TITLE
Debian packages for work with XML in HTTPMOD / MQTT Port

### PR DESCRIPTION
**Pull request by cheanrod**
I added several debian perl packages that can be used in HTTPMOD in order to deal with XML parsing.

Also added an EXPOST for MQTT2SERVER (1883).

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -194,7 +199,7 @@ RUN chmod 755 /*.sh /usr/local/bin/speedtest-cli
 
 VOLUME [ "/opt/fhem" ]
 
-EXPOSE 7072 8083
+EXPOSE 7072 8083 1883
```
Comment (jpawlowski) [inline Dockerfile]: Extending EXPOSE is not needed here as one cannot be sure that a user is actually using it. It is also not required for the container to work properly as technically this has nothing to do with making ports available outside of the host network. Kindly revert this change.

File: Dockerfile [hunk 2]
```diff
@@ -167,6 +167,11 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get update \
         libxml-simple-perl \
         libnet-sip-perl \
         libxml-stream-perl \
+        libhtml-strip-perl \
```
Comment (jpawlowski) [inline Dockerfile]: Kindly ensure proper sorting of the added lines in order to keep the clear structure and better overview.

File: Dockerfile (hunk 1 above)
Comment (cheanrod) [reply Dockerfile]: I agree with your comment on EXPOSE.
